### DISCUSSION TITLE
Message attribute when using validate function.

### DISCUSSION THREAD

**Discussion by jjwwiodyne** [react-hook-form/react-hook-form#2829] [Q&A] [answered]
How do you set the error message attribute on `require()` when using a validation function? 

Normally, for the other validation types I'd pass a `{value: <function here>, message: "Some error"}` dictionary, but that doesn't seem to work for `validate`:
```
register({
            validate:  {
                 value:(value) => {
                      return value === watch('user_password');
                 },
                message: "Passwords don't match"
            }
          })
```

Thank you in advance.

**Comment by bluebill1049** [comment, marked as answer]
https://react-hook-form.com/api#register

<img width="1661" alt="Screen Shot 2020-09-05 at 10 40 39 am" src="https://user-images.githubusercontent.com/10513364/92293109-3ef17100-ef64-11ea-94ea-9135da96eb74.png">

**Reply by jjwwiodyne** [reply]
Thanks Bill! I get a different docs page for register:

![image](https://user-images.githubusercontent.com/52980818/92293435-c39aab80-eee0-11ea-88f7-429d037826f4.png)

**Reply by bluebill1049** [reply]
scroll up you miss the toggle button for message part.

**Comment by jjwwiodyne** [comment]
Awesome. Thank you.
